app/models/response_snapshot.rb

```diff
@@ -0,0 +1,15 @@
+# Works same as Response.
+#
+# Uses the results stored in its ResponseFinished event instead of getting info realtime from R.
+class ResponseSnapshot < Response
```

Comment:
RFC:

Nu heeft `ResponseSnapshot` (response die resultaten uit DB haalt) `Response` ("realtime" response die resultaten uit R haalt) als base class, en overschrijft attributen die Response zelf ook al definieert.

Vinden jullie het kwa OOP-stijl netter om een `AbstractResponse` bovenaan de hierarchie te hebben (die geen domain_results, estimate etc implementeert), met `RealtimeResponse < AbstractResponse` en `ResponseSnapshot < AbstractResponse`, of mag het zo blijven?